05.ls/ls.rb

```diff
@@ -107,7 +107,7 @@ def output_total(detail_files)
 end
 
 def output_detail(detail_files)
-  col_width = calc_detail_col_width(detail_files)
+  col_width = calc_detail_col_widths(detail_files)
```

Comment:
```suggestion
  col_widths = calc_detail_col_widths(detail_files)
```

のように左右で名前を揃えた方がベターですね。